PS-Policy.md

```diff
@@ -24,4 +24,25 @@ See the **_Applicability_** section of the [GSA IT Security Policy](http://www.g
 
 For information on roles and responsibilities, management commitment, coordination among organizational entities, compliance, reviews, and updates please see the [Technology Transformation Service's (TTS) Common Control Policy](https://github.com/18F/compliance-docs/blob/master/TTS-Common-Control-Policy.md).
 
-## Procedures
\ No newline at end of file
+## Procedures
+
+Whenever Cloud Operations, or any other team or individual, discovers that information security policies or procedures have been violated, they must immediately follow the 18F incident notification procedures (which notifies GSA Information Security teams) [LINK] and notify the System Owner, information system Authorizing Official, and the individual's direct supervisor via GSA email, separately. All notifications must occur within 24 hours of detecting a policy or procedure violation.
```

Comment:
For "information security policies or procedures", can we be specific here, saying something like "GSA, 18F, or cloud.gov information security policies or procedures"? That would help me as a reader figure out when to apply this procedure.

It would help to be clear about whether the person should specifically follow https://docs.cloud.gov/ops/security-ir/ or directly skip to https://github.com/18F/security-incidents#process (which probably depends on whether they violated a cloud.gov-specific policy/procedure or an 18F/GSA one).